vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -950,6 +951,56 @@ public void failure(VimeoError error) {
 
     // </editor-fold>
 
+    // -----------------------------------------------------------------------------------------------------
+    // Documents
+    // -----------------------------------------------------------------------------------------------------
+    // <editor-fold desc="Documents">
+
+    /**
+     * An asynchronous request to get a {@link Document} representing the Vimeo Terms of Service
+     *
+     * @param callback the {@link ModelCallback} to be invoked when the request finishes
+     * @return a {@link Call} with the {@link Document} type. This can be used for request
+     * cancellation.
+     */
+    @NotNull
+    public Call<Document> getTermsOfService(@NotNull ModelCallback<Document> callback) {
+        return getDocument(Vimeo.ENDPOINT_TERMS_OF_SERVICE, callback);
+    }
+
+    /**
+     * An asynchronous request to get a {@link Document} representing the Vimeo Privacy Policy
+     *
+     * @param callback the {@link ModelCallback} to be invoked when the request finishes
+     * @return a {@link Call} with the {@link Document} type. This can be used for request
+     * cancellation.
+     */
+    @NotNull
+    public Call<Document> getPrivacyPolicy(@NotNull ModelCallback<Document> callback) {
+        return getDocument(Vimeo.ENDPOINT_PRIVACY_POLICY, callback);
+    }
+
+    /**
+     * An asynchronous request to get a {@link Document} representing the Vimeo Payment Addendum
+     *
+     * @param callback the {@link ModelCallback} to be invoked when the request finishes
+     * @return a {@link Call} with the {@link Document} type. This can be used for request
+     * cancellation.
+     */
+    @NotNull
+    public Call<Document> getPaymentAddendum(@NotNull ModelCallback<Document> callback) {
+        return getDocument(Vimeo.ENDPOINT_PAYMENT_ADDENDUM, callback);
+    }
+
+    @NotNull
+    private Call<Document> getDocument(@NotNull String uri, @NotNull ModelCallback<Document> callback) {
```

Comment:
I think this method could be public - it just future proofs us in case consumers want to request documents at other URIs